theano/sandbox/blocksparse.py

```diff
@@ -56,7 +56,7 @@ def make_node(self, o, W, h, inputIdx, outputIdx):
         outputIdx = theano.tensor.as_tensor_variable(outputIdx)
 
         if o.ndim != 3:
-            raise TypeError('The output o must be a 3D tensor')
+            raise TypeError('The output o must be a 2D tensor')
```

Comment:
o is a 3D vector. The bias (2D) has been copied several times in it beforehand though.